src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -1002,6 +1032,9 @@ bool Pane::_CanSplit(SplitState splitType)
     const Size actualSize{ gsl::narrow_cast<float>(_root.ActualWidth()),
                            gsl::narrow_cast<float>(_root.ActualHeight()) };
 
+    const Size desiredSize = _root.DesiredSize();
+    desiredSize;

```

Comment:
I don't like this at all: are we using a getter for a side effect?

If so, `(void)_this.Thing()` with a comment explaining it

Reply:
Oh no, that's dead code